app/pdfs/timelogs_pdf.rb

```diff
@@ -0,0 +1,21 @@
+require 'prawn'
+require 'prawn/table'
+class TimelogsPdf < Prawn::Document
+
+  def initialize(timelogs)
+    super()
+    @timelogs = timelogs
+    fill_document
+  end
+
+  def fill_document
+    text "Timelogs\n" + "Owner: " + User.find(@timelogs.first.user_id).email.to_s
```

Comment:
please, extract this into a separate method. Each method should do one small thing.